### ISSUE TITLE
Review annotations to GO:0098772 molecular function regulator

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2827]
Hello, 

We will make the term "GO:0098772 molecular function regulator" 'do not annotate', see https://github.com/geneontology/go-ontology/issues/18868. 

Please try to annotate to a more specific child term, or request a new term.   

Annotations are here: 
https://docs.google.com/spreadsheets/d/1q0ZqlK9r1GfjWq7Yxoggln8O0RQhXOoUgSlsWcNVxnw/edit#gid=0

BHF-UCL - 2 annotations
ComplexPortal - 1 annotation
MGI - 4 annotations
SGD - 1 annotation
UniProt - 1 annotation
WB - 1 annotation

Thanks, Pascale

**Comment by ValWood** [comment]
Isn't this recommended to use in Noctua when you don't know what the activity is?

**Comment by RLovering** [comment]
BHF-UCL complete

**Comment by pgaudet** [comment]
@ValWood 

> Isn't this recommended to use in Noctua when you don't know what the activity is?

We don't recommend using a MF if we don't know what the MF is. You can use BP 'regulation of x pathway', for example.

**Comment by ValWood** [comment]
Ah no I remember, it's just "molecular function" OK ignore;)

**Comment by srengel** [comment]
SGD done.

**Comment by bmeldal** [comment]
CP removed, also in CPX-141 (mouse)

**Comment by ukemi** [comment]
Some of these are coming from GO-CAM models. We should have put a plan in place for blocking those before opening this ticket.

**Comment by ukemi** [comment]
@balhoff will these be suppressed once the tag is added?

**Comment by balhoff** [comment]
@ukemi I don't think so; they will probably just trigger a GO-RULES error somewhere down the line.

**Comment by ukemi** [comment]
Then at this point we are stuck.

**Comment by pgaudet** [comment]
@balhoff Can we not block inferences or export to 'do not annotate' terms ?

**Comment by vanaukenk** [comment]
WB annotation is gone.  This may have been coming as an inference, no longer being made, from a GO-CAM model.

**Comment by pgaudet** [comment]
Only 4 pombase annotations left - 

PomBase:SPCC777.08c	Protor homolog, Bit61	PomBase	EXP	PMID:21035342	20210129
PomBase:SPAC926.03	myosin II regulatory light chain Rlc1	PomBase	EXP	PMID:19570908	20210522
PomBase:SPAC4C5.02c	Rab family GTPase Ryh1	PomBase	EXP	PMID:21035342	20210129
PomBase:SPAC27F1.02c	tropomyosin	PomBase	IDA	PMID:12796476	20200317

@ValWood @mah11

**Comment by pgaudet** [comment]
The do not annotate flag is on these terms; you should bet getting errors for these annotations.

**Comment by hdrabkin** [comment]
GO:0098772 has no annotations in MGI;

**Comment by mah11** [comment]
The PomBase annotations are waiting for the resolution of two tickets, https://github.com/geneontology/go-ontology/issues/18749 and https://github.com/geneontology/go-ontology/issues/19107, where Val has asked about where Val has asked about which terms to use.

**Comment by pgaudet** [comment]
For PomBase:SPAC926.03 myosin II regulatory light chain Rlc1 PomBase EXP PMID:19570908 20210522

We have created regulator of motor activity (although there is some argument in https://github.com/geneontology/go-annotation/issues/3819  as to whether the RLC chain is really regulatory, or if it's just necessary and part of the activity)

**Comment by ValWood** [comment]
PomBase:SPCC777.08c Protor homolog, Bit61 PomBase EXP PMID:21035342 20210129
PomBase:SPAC4C5.02c Rab family GTPase Ryh1 PomBase EXP PMID:21035342 20210129

are a problem.
We know the directly regulate on tor1 (bit61//SPCC777.08c) and  bit61 (ryh1/SPAC4C5.02c)
as part of TORC2 signalling but I don't know what the actual activities are.

Without them my signalling pathway model model will be incomplete.

**Comment by ValWood** [comment]
I thought that for GO-cam using "molecular function/molecular function regulator"  was allowed. Did this change?
@vanaukenk

**Comment by ValWood** [comment]
> PomBase:SPAC27F1.02c tropomyosin PomBase IDA PMID:12796476 20200317

This will be dealt with when we have ways to annotate the MFs of cytoskeletal proteins- lots of open tickets about this where often MFs are in the BP ontology.

I'm still a little confused  about blocking the "molecular function regulator'  term. From reading the thread, it seems these annotations can be made in Noctua, but are not exported into GO. This doesn't really make sense to me. If they are available to Noctua they should be available for normal GO annotation. We are using this to build GO-cams.
